apps/frontend/example.env

```diff
@@ -26,4 +26,8 @@ AZURE_TENANT_ID=
 NEXT_PUBLIC_BACKEND_PROXY_PATH=/api
 
 # URL used in next.config.js to set resources for image loading
-CDN_BASE_URL=http://localhost:3000/api/uploads
\ No newline at end of file
+CDN_BASE_URL=http://localhost:3000/api/uploads
+
+#Google Recaptcha
+NEXT_PUBLIC_RECAPTCHA_SITE_KEY=6LdxdzgpAAAAACNtxhxCjOH8BbNh87Ua9m_iBuOG
+NEXT_PUBLIC_RECAPTCHA_SECRET_KEY=6LdxdzgpAAAAAM91BOAuPw7fJXLcKkWFwcI7-a_H
```

Comment:
perfixing env variables with `NEXT_PUBLIC` makes them accessible from the frontend. `RECAPTCHA_SECRET_KEY` should be accessible only from the backend, so please remove the prefix.